### PR TITLE
Feat/dennis/landing

### PR DESCRIPTION
**Pull request by Dennull**
Made a basic landing page, still a WIP

### CODE REVIEW

**Comment by RatikKapoor** [comment]
That looks amazing @Dennull!! I'm just going to hold off of reviewing this until #40 gets merged in. After that you can rebase and push your branch on top of main and we'll give it a review!

**Review by rjb75 — CHANGES_REQUESTED** [review]
@Dennull Looks good, just if you wanted to rebase and then commit first like @RatikKapoor mentioned above that would be great!

**Review by RatikKapoor — APPROVED** [review]
🚀

File: src/components/LandingFeatureBox.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                </IonList>
+            </IonItem>
+            <IonItem className="FeatureBoxRight" lines="none">
+                <IonItemDivider className="ImagePlaceHolder"></IonItemDivider>
+            </IonItem>
+        </>
+    );
+};
+
+export default LandingFeatureBox;
```
Comment (lemohammed) [inline src/components/LandingFeatureBox.tsx:46]: This is pretty good!

File: src/components/LandingInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <div className="placeholderBlock">Placeholder</div>
+            <div className="placeholderBlock">Placeholder</div>
+            <div className="placeholderBlock">Placeholder</div>
+            <div className="placeholderBlock">Placeholder</div>
+            <div className="placeholderBlock">Placeholder</div>
+        </div>
+    );
+};
+
+export default LandingInfo;
```
Comment (lemohammed) [inline src/components/LandingInfo.tsx:54]: This needs to be centred. Should we just commit this to main and raise an issue for later?

**Review by lemohammed — CHANGES_REQUESTED** [review]

File: src/components/LandingInfo.tsx (hunk 2 above)
Comment (RatikKapoor) [reply src/components/LandingInfo.tsx:54]: @lemohammed Yeah that's what I was thinking. Better to have the bulk of the code in already but then later get it production-ready

File: src/components/LandingInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './LandingInfo.scss';
+import { IonButton } from '@ionic/react';
+
+const LandingInfo: React.FC = () => {
+    return (
+        <div>
```
Comment (RatikKapoor) [inline src/components/LandingInfo.tsx:7]: Just waiting on this to be changed to Ionic components and we'll go ahead and merge

**Review by RatikKapoor — CHANGES_REQUESTED** [review]

**Review by rjb75 — APPROVED** [review]

**Review by RatikKapoor — APPROVED** [review]
🚀

**Review by lemohammed — APPROVED** [review]

**Pull request merged**